src/graphics/hud_manager.rs

```diff
@@ -19,29 +29,40 @@ pub struct HUDManager {
     /// The rects that are always present on the screen
     base: Vec<RectVertexAttr>,
 
+    help_menu_data: HelpMenuData,
     help_menu: HelpMenu,
     show_help: bool,
 
+    debug_menu_data: DebugMenuData,
```

Comment:
I think the same reason justify to remove this attribute. An attribute that is not used must be removed.